cores/esp32/HardwareSerial.cpp

```diff
@@ -477,6 +510,12 @@ void HardwareSerial::setDebugOutput(bool en) {
   if (_uart == 0) {
     return;
   }
+#if (SOC_UART_LP_NUM >= 1)
+  if (_uart_nr == LP_UART_NUM_0) {
```

Comment:
```cpp
if (_uart_nr >= SOC_UART_HP_NUM) {
```
Will handle cases where LP uarts are more than one (future)

Reply:
It can be done like that for future extension, although there no single applicable case.
I'll review it.

Reply:
Ok. Done.